attachments_component/admin/tmpl/attachments/default.php

```diff
@@ -31,7 +33,14 @@
 
 ?>
 <form action="<?php echo Route::_('index.php?option=com_attachments'); ?>" method="post" name="adminForm" id="adminForm">
-<?php echo $this->loadTemplate('filter');?>
+<?php if ( $this->editor ) : ?>
+		<button onclick="insertAttachmentsIdToken(jQuery, '<?php echo $this->editor ?>');"><?php echo "{attachmentid id=xx,yy}"; ?></button>
```

Comment:
The button text could be a bit better so the user can understand that he/she should press it after selecting the attachments. How about "Add selected attachments" instead of "{attachmentid id=xx,yy}"? Also, we could add a note before the button so that the user knows why the list is so small or probably empty. Something along the lines of "This is a list of attachments already added the the article/category. Select which one(s) you want to insert using the token and press the 'Add selected attachments' button."